api/dump.go

```diff
@@ -83,13 +83,19 @@ func (s *service) prepareDump(ctx context.Context, state *task.ProcessState, arg
 			break
 		}
 	}
-	opts.ShellJob = proto.Bool(isShellJob)
+	opts.ShellJob = proto.Bool(isShellJob || args.Stream)
```

Comment:
In the [examples provided](https://github.com/checkpoint-restore/criu-image-streamer), `--shell-job` is always provided to criu so I assumed so.

Reply:
Interesting! I think this might be just because their examples are all shell jobs and it's unrelated to criu image streamer specifically. But docs are unclear on this, we could try the streamer without the opt

Reply:
You were right, I reverted it.